microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -33,35 +91,79 @@ To submit suggestions and feedback, enter an [Issue about the Add-ons API](https
 
 
 <!-- ====================================================================== -->
-## Before you begin
+## Enabling the Update REST API at Partner Center
+
+To use the Microsoft Edge Add-ons API, you first need to enable the API for your project in the Microsoft Partner Center by creating API credentials, as follows:
+
+##### [v1.1](#tab/v1-1)
+
+1. At your [Partner Center developer dashboard](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd), sign in to the account that you used to publish an add-on.
+
+1. Under the **Microsoft Edge** program, select **Publish API**.
+
+1. On the **Publish API** page, click the **Create API credentials** button.  This step may take a few minutes to finish.
 
-To use the Microsoft Edge Add-ons API, you need to enable the API for your project in the Microsoft Partner Center, by creating API credentials. Use the following steps to create API credentials.
+   The API credentials have now been created; you've enabled or renewed the API.  The following items are now displayed on the **Publish API** page:
+   * **Client ID**
+   * **API key**
+   * **Expiry date**
 
-1. Visit Microsoft Partner Center and sign in to the account that you used to publish an add-on.
+   ![The 'Publish API' page at Partner Center after clicking 'Create API credentials', now showing Client ID and API key](./using-addons-api-images/create-api-credentials-button.png)<!-- todo: add/use new png showing v1.1.  What's the title of the page that contains the UI control to select v1 vs v1.1?  What's the label of the UI control?  Is the UI control an Option button, or a Button? -->
```

Comment:
provide screenshot png of Partner Center showing v1.1 UI/fields, equiv to the v1 png that's in v1 tab.
* Whole window (not fullscreen, not portion of window).
* Make window as narrow as feasible, while still looking normal - to get crisp result in rendered article.

@yvsubhash @bhuvanapriyap 

rendered article section, v1 tab's png:
https://review.learn.microsoft.com/microsoft-edge/extensions-chromium/publish/api/using-addons-api?branch=pr-en-us-3269&tabs=v1#enabling-the-update-rest-api-at-partner-center

Reply:
added 2 pngs